funnel/templates/event.html.jinja2

```diff
@@ -1,7 +1,8 @@
 {% extends "layout.html.jinja2" %}
 {% from "macros.html.jinja2" import rsvpform, rsvpscript %}
 {% from "baseframe/forms.html.jinja2" import renderfield %}
-{% block title %}{{ project.title }}{% endblock %}
+{% block title %}Setup events{% endblock %}
+{% set title_suffix = project.title %}
```

Comment:
I think this needs to be right below extends, or maybe above it. Can you confirm that it works in this position? If so, it's okay.

Reply:
Yes, it works